app/models/time_reg.rb

```diff
@@ -25,16 +26,25 @@ def active?
 
   def toggle_active
     if active?
-      worked_minutes = (Time.now.to_i - start_time.to_i) / 60
-      self.minutes = [ minutes + worked_minutes, MINUTES_IN_A_DAY ].min
-      self.start_time = nil
+      deactivate
     else
       self.start_time = Time.now
     end
 
     save!
   end
 
+  def deactivate!
```

Comment:
i think a name like `stop` sounds more suitable to the actual action happening and self explanatory, `deactivate` seems a bit different to what's happening here. What do you think?

I'm just being nitpicky here

Reply:
Yes! `stop` sounds much better. :smiley: